database/similarProperties.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+const db = require('./index.js');
+mongoose.Promise = global.Promise;
+
+const similarPropertiesSchema = new mongoose.Schema({
+  propertyID: Number,
+  photos: [String],
+  location: String,
+  typeOfRoom: String,
+  totalBeds: Number,
+  nameOfProperty: String,
+  pricing: Number,
+  rating: Number,
+  reviews: Number
```

Comment:
A small suggestion to make it as close to original version as possible - I looked at your module on Airbnb and I see that properties in your module that are posted by superhost have a copy 'Superhost' shown right on top of propertyName/ next to typeOfRoom. If you are planning to add this, you will need host ID field in your schema so that you can get the host info and check whether to show this copy or not.